site/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "vocs dev",
-    "build": "vocs build",
+    "build": "./retry-build.sh",
```

Comment:
this is sick btw. means our general CI step should be a bit faster since we don't need to `retry` the root `yarn build` thanks for doing this!